packages/editor/src/browser/editor-preferences.ts

```diff
@@ -458,6 +458,21 @@ export const editorPreferenceSchema: PreferenceSchema = {
             'type': 'boolean',
             'description': 'Reveal first change.',
             'default': true
+        },
+        'files.eol': {
+            'type': 'string',
+            'enum': [
+                '\n',
```

Comment:
for some reasons only `auto` is available in the preference tree :-(

Reply:
Should be solved separately. Filed https://github.com/theia-ide/theia/issues/4087.